### ISSUE TITLE
UI system and guidlines

### ISSUE THREAD

**Issue by DrLuk** [EveronLife/EveronLife#70]
### Summarize your proposed improvement

Formation of a UI "taskforce" to work out a modular, simple and future proof UI system. This should make sure that everything added to the project fits together nicely. It needs to look appealing but also consist of a simple but effective, modular structure so we can expand and use it for everything. Some basics are: defined colors, a modular interaction menu, uniform sizes of images/icons/textboxes/etc.

### Why is this needed?

In-game menus in Arma 3 are a mess, they almost always look ugly and in today's standarts: obsolete. Creating a solid UI system as a base that we can use for all future updates to build on makes the most sense in the early stages of development. Changing such a system later is going to require quite some work, so why don't do it the right way from the start.

**Comment by LuxPoseidon** [comment]
I agree mostly with that, it defiantly has a good structure but I do feel it is missing (or at least stuff has no documentation on how to do it yet) some things. There are a few cases I have run into that make it seem like custom Widgets are needed.

**Comment by Tomboru** [comment]
I would suggest one thing.
It has to be compatible with the console so it's still Cross-platform.

We have all the tools for it, so we should use them correctly.
I know that HTML UI looks probably better or is more functional.
But then its just not Cross-platform

**Comment by DrLuk** [comment]
> It has to be compatible with the console so it's still Cross-platform.
Exactly my point. We need to think of players that use controllers. Especially with things like selecting amounts of an item etc.
